CARET_trace/src/ros_trace_points.cpp

```diff
@@ -51,6 +52,92 @@
 std::unique_ptr<std::thread> trace_node_thread;
 thread_local bool trace_filter_is_rcl_publish_recorded;
 
+using std::string;
+using std::vector;
+static bool ignore_rcl_timer_init = false;
```

Comment:
Don't we need `thread_local` here? like line 53?
(I'm just asking because L53 has `thread_local` )

Reply:
I tried it at first, but I couldn't use thread_local.
std::this_thread::get_id() is different for ros_trace_rcl_timer_init() and is_ros2_launch_command().